source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -170,6 +171,7 @@ namespace AdaptiveCards::Rendering::Uwp
         {
             if (pixelWidth)
             {
+                SetRasterizedPixelWidth(imageSource, pixelWidth);
```

Comment:
>SetRasterizedPixelWidth [](http://example.com/codeflow?start=16&length=23)

Is it valid for the SVG to provide width but not height (and vice versa)?

Reply:
Yes, that's a valid use case. SvgImageSource will inherit the non-set variable from the parent.